pom.xml

```diff
@@ -160,10 +156,7 @@
 						</goals>
 						<phase>process-sources</phase>
 						<configuration>
-							<roots>
```

Comment:
Why was this removed?

Reply:
The default just takes all sources. The root-tag was a copy-paste left-over.